maven-repository-provisioner/src/main/java/com/simpligility/maven/provisioner/Configuration.java

```diff
@@ -109,6 +109,18 @@ public class Configuration {
             arity = 1)
     private Boolean verifyOnly = false;
 
+    @Parameter(
+            names = {"-pd", "-parallelDeploy"},
+            description = "Whether or not to deploy artifacts in parallel.",
```

Comment:
Activate parallel deployment of artifacts.
```suggestion
            description = "Whether or not to deploy artifacts in parallel. Defaults to false.",
```